subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsTask.kt

```diff
@@ -120,8 +129,15 @@ abstract class InstrumentationTestsTask @Inject constructor(
     private val verdictFile = objects.fileProperty().convention(output.file("verdict.json"))
 
     @get:Internal
-    override val verdict: String
-        get() = verdictFile.asFile.get().readText()
+    override val verdict: SpannedString
+        get() {
+            val verdictRaw = verdictFile.asFile.get().reader()
+            val verdict = gson.fromJson<Verdict>(verdictRaw)
+            return link(
```

Comment:
This is how the link creation looks on the user's side

Reply:
i like not statically imported `SpannedString.link()` more, easier to read where it comes from

Reply:
Will you describe the usage of the plugin in documentation?

Reply:
I see it selfdocumented because I wrote it
Could you give me an example of cases you want to see in the documentation

Reply:
I will create a task for this